aws-android-sdk-cognitoauth/src/main/java/com/amazonaws/mobileconnectors/cognitoauth/activities/CustomTabsManagerActivity.java

```diff
@@ -0,0 +1,127 @@
+package com.amazonaws.mobileconnectors.cognitoauth.activities;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.util.Log;
+
+import com.amazonaws.mobileconnectors.cognitoauth.AuthClient;
+
+import java.util.logging.Logger;
+
+public class CustomTabsManagerActivity extends Activity {
+    static final String CUSTOM_TABS_LAUNCHED_KEY = "customTabsLaunched";
+    static final String CUSTOM_TABS_INTENT_KEY = "customTabsIntent";
+
+    private boolean customTabsLaunched = false;
+    private Intent customTabsIntent;
+
+    /**
+     * Creates an intent to start an OAuth2 flow in Chrome custom tabs.
+     * @param context the package context for the app.
+     * @param customTabsIntent the intent for the Chrome custom tab.
+     */
+    public static Intent createStartIntent(
+            Context context,
+            Intent customTabsIntent
```

Comment:
Can we annotate the the parameters and return values with `@NonNull` and `@Nullable` for every publicly exposed methods?

Reply:
nit: not worth the efforts in the v2 SDK